src/category_theory/functor_category.lean

```diff
@@ -5,6 +5,19 @@ Authors: Tim Baumann, Stephen Morgan, Scott Morrison, Floris van Doorn
 -/
 import category_theory.natural_transformation
 
+/-!
+# The category of functors and natural transformations between two fixed categories.
```

Comment:
This is a bit long of a name! Also, maybe that's just me but "category of functors and natural transformations" makes me think that the natural transformations are objects. Aren't "Functor category" or "Category of functors" (+ eventually the fixedness clarification) well-established names?

Reply:
I think "the category of [objects] and [morphisms]" is a commonly used naming pattern for categories.